coopersystem-fsd/pcc
# Adjusted content and languages

## Description

- Add more languages
- resize the space between title and content 

## Steps to test

1. See an event single page;
2. See the edit page event and see the languages

## Additional information
